examples/lib/defaultHtmlStepUi.js

```diff
@@ -24,6 +24,9 @@ function DefaultHtmlStepUi(_sequencer, options) {
   function onSetup(step, stepOptions) {
     if (step.options && step.options.description)
       step.description = step.options.description;
+    
+    let stepDocsLink = '';
+    if (step.moduleInfo) stepDocsLink = step.moduleInfo['docs-link'] || '';
```

Comment:
Hi all!!!  Should we default to https://github.com/publiclab/image-sequencer/blob/main/docs/MODULES.md do you think?

Reply:
We can do that! Does the first step have any docs anyway?

Reply:
Maybe not but it could have a brief explanation maybe, we can add it later. Thanks!!!

Reply:
@vivek-30 want to try changing the default link to https://github.com/publiclab/image-sequencer/blob/main/docs/MODULES.md?

Reply:
@HarshKhandeparkar and @jywarren  i have fixed another issue #1724 , and also tested it. so  can i raise it with some different branch ?

Reply:
Yes

Reply:
Use a different branch and a separate PR.